deployments/qat_plugin/base/daemonset-kernel/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
+  - intel-qat-kernel-plugin.yaml
```

Comment:
I think this is enough for the `-mode kernel` deployment (no need for the overlays for it)